os/linux/weston/plugin/compositor-iahwc.c

```diff
@@ -1217,6 +1216,7 @@ static void iahwc_assign_planes(struct weston_output *output_base,
       struct weston_surface *es = ev->surface;
       es->keep_buffer = false;
 
+#ifdef IAHWC_DISABLE_OVERLAYS
```

Comment:
hmm sorry this should be #ifndef ?

Reply:
that code will be compiled only when IAHWC_DISABLE_OVERLAYS is set.

Reply:
but this will disable overlays usage by default. The code needs to be executed for using overlays right

Reply:
No, this code is required only when we want to go for single plane rendering from weston.